src/java.naming/share/classes/javax/naming/ldap/package-info.java

```diff
@@ -173,45 +171,45 @@
  *     }
  *     ...
  * }
- * </pre></blockquote>
+ * }
  * <p>
  * When a service provider receives response controls, it uses
- * the <code>ControlFactory</code> class to produce specific classes
- * that implement the <code>Control</code> interface.
+ * the {@code ControlFactory} class to produce specific classes
+ * that implement the {@code Control} interface.
  * <p>
  * An LDAP server can send back response controls with an LDAP operation
  * and also with enumeration results, such as those returned
  * by a list or search operation.
- * The <code>LdapContext</code> provides a method (<code>getResponseControls()</code>)
+ * The {@code LdapContext} provides a method ({@code getResponseControls()})
  * for getting the response controls sent with an LDAP operation,
- * while the <code>HasControls</code> interface is used to retrieve
+ * while the {@code HasControls} interface is used to retrieve
  * response controls associated with enumeration results.
  * <p>
  * For example, suppose an LDAP server sends back a "change ID" control in response
- * to a successful modification. It would supply a class <code>ChangeIDControl</code>
+ * to a successful modification. It would supply a class {@code ChangeIDControl}
  * so that the application can use this feature.
  * An application would perform an update, and then try to get the change ID.
- * <blockquote><pre>
+ * {@snippet :
  * // Perform update
  * Context ctx = ectx.createSubsubcontext("cn=newobj");
  *
  * // Get response controls
  * Control[] respCtls = ectx.getResponseControls();
  * if (respCtls != null) {
  *     // Find the one we want
- *     for (int i = 0; i &lt; respCtls; i++) {
+ *     for (int i = 0; i < respCtls.length; i++) {
```

Comment:
This is the only change that isn't a direct 1:1 conversion, I hope this ok.

Reply:
This seems reasonable for what is an obvious but minor bug

Reply:
The change looks correct to me too.